content/post/11-Eliminating-Distractions/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Eliminating Distractions"
+date: 2010-11-13T15:56:33-07:00
+draft: true
+description: This is a description
+categories:
+- life
+- travel
```

Comment:
If you look at the bottom-right hand corner of the original blog post on my current site, it shows the only category as "zen." Can you add that using all lowercase? 

![image](https://user-images.githubusercontent.com/4295964/50741603-88505280-11bc-11e9-81b7-005a5af91dcc.png)

So, this should be changed to:

```yaml
categories:
- zen
```